src/main/Extensions/VSCode/VSCodeExtension.ts

```diff
@@ -235,10 +237,18 @@ export class VSCodeExtension implements Extension {
             this.getSettingDefaultValue("showPath"),
         );
 
-        const searchResultItems = this.recents.map((recent) => this.getSearchResultItem(recent, template, showPath));
-
         searchTerm = searchTerm.replace(this.getPrefix() + " ", "").trim();
 
+        if (!isPath(searchTerm)) {
+            return this.getRecentSearchResults(searchTerm, template, showPath);
+        } else {
+            return this.getFilesystemSearchResults(searchTerm, template);
+        }
```

Comment:
Feel free to leave it as it is, I'm nit-picking here, but how about turning this into one single return statement? Don't you think that's much nicer to read?

```typescript
return isPath(searchTerm)
    ? this.getFilesystemSearchResults(searchTerm, template)
    : this.getRecentSearchResults(searchTerm, template, showPath);
```

Reply:
Personally I find brackets much nicer you end up just refactorying turnaries like that if you need to add something to one of the branches.

If its a simple value expression that doesnt line wrap I'll use them but other than that I usually stick with if else

Reply:
Ok, valid point :)